cbw_api_toolbox/cbw_api.py

```diff
@@ -30,10 +31,27 @@
 class CBWApi: # pylint: disable=R0904
     """Class used to communicate with the CBW API"""
 
-    def __init__(self, api_url, api_key, secret_key, verify_ssl=False):
-        self.api_url = api_url
-        self.api_key = api_key
-        self.secret_key = secret_key
+    def __init__(
+        self,
+        api_url=None,
+        api_key=None,
+        secret_key=None,
+        verify_ssl=False,
+    ):
+
+        try:
+            self.api_url = api_url if api_url is not None else environ["CBW_API_URL"]
+            self.api_key = api_key if api_key is not None else environ["CBW_API_KEY"]
+            self.secret_key = (
+                secret_key if secret_key is not None else environ["CBW_SECRET_KEY"]
+            )
```

Comment:
Here I checked if the 3 variables were explicitly not equal to `None` instead of checking whether their values where `False` after being converted to a boolean. This allow the use of the empty string value (`''`) during tests.